Engine/src/Application.bf

```diff
@@ -155,6 +157,7 @@ namespace SteelEngine
 			return true;
 		}
 
+		protected virtual void OnUpdate(float dt) { }
```

Comment:
Maybe instead of passing the deltatime down the chain, just let the user get it themselves from Time.DeltaTime?

Reply:
Yeah, probably. I just saw that Update method had this parameter so I kinda rolled with that.

Reply:
I think it's for the better to remove it ;) From both the OnUpdate and update or wherever you found it. Now, since we have Time.DeltaTime there is no need for that.